minio/minio
# fix: simplify APIEndpoints() usage

## Description
fix: simplify APIEndpoints() usage

## Motivation and Context
improvements include

- skip IPv6 correctly
- do not set a default value for
  MINIO_SERVER_URL, let it be
  configured if not use local IPs

Bonus:

- In healing return error from listPathRaw()
- update console to v0.8.3

## How to test this PR?
Nothing special set environment variables like and see what is printed on the banner
```
MINIO_SERVER_URL
MINIO_BROWSER_REDIRECT_URL
```

## Types of changes

## Checklist:
